cores/esp8266/PolledTimeout.h

```diff
@@ -214,8 +226,8 @@ class timeoutTemplate
   bool checkExpired(const timeType internalUnit) const
   {
     // canWait() is not checked here
-    // returns "can expire" and "time expired"
-    return (!_neverExpires) && ((internalUnit - _start) >= _timeout);
+    // returns "can expire" and "oneshot not expired" and "time expired"
```

Comment:
Would the comment rather be:
> `"already verified to be expired" - or - "can expire" and "time expired"

Why is `_oneShotExpired` set in `::expiredOneShot()` and not in this function ?